src/firecracker/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "firecracker"
-version = "0.23.0"
+version = "0.24.0"
```

Comment:
Please also run a `cargo check`, so that we also update the lockfiles

Reply:
Yeah, actually I missed to include cargo.lock in any of the commits.

Reply:
Fixed.